crates/wasmtime/src/runtime/func.rs

```diff
@@ -1445,7 +1452,8 @@ impl Func {
     /// | `(ref nofunc)`                            | `NoFunc`                              |
     /// | `v128`                                    | `V128` on `x86-64` and `aarch64` only |
```

Comment:
While you're here mind removing this clause about aarch64/x86_64? (it's available on all platforms now)

Reply:
Ah I had no other comments so gonna go ahead and merge